crypto/ec/ecp_nistp521.c

```diff
@@ -131,6 +131,7 @@ static const felem_bytearray nistp521_curve_params[5] = {
 #define NLIMBS 9
 
 typedef uint64_t limb;
+typedef limb limb_aX __attribute((__aligned__(1)));
```

Comment:
As above (do we need to support alternative compiler extensions for unaligned accesses?)

Reply:
No see my other comment.  This does really not look like we will have to.